src/client/_clientwindow.py

```diff
@@ -870,12 +874,25 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
+        text_of_autojoin_settings_dialog = """
+        Autojoin channels list : """ + str(autojoin_channels_list) + """
+        Enter the list of channels you want to join, separated by ;
+        For example: #poker;#newbie
+        To disable autojoin, leave box empty and press ok
+        """
+        channels_input_of_user, ok = QInputDialog.getText(None, 'Set autojoin channels',text_of_autojoin_settings_dialog, QLineEdit.Normal, "")
+        if ok:
+            Settings.set('chat/auto_join_channels', channels_input_of_user.split(';'))
```

Comment:
I think it would be a good idea to strip() the string list so nobody gets confused when they do '#channel1 ; #channel2'. If we really want to keep it a one-liner, we can even do a list(map(str.strip, channels.split(';'))).

Reply:
ok done